doc/bootstrap.rst

```diff
@@ -51,28 +65,29 @@ This command does a few things:
 Interacting with the cluster
 ============================
 
-You can easily start up a container that has all of the Ceph packages
-installed to interact with your cluster::
+To interact with your cluster, start up a container that has all of 
+the Ceph packages installed::
 
-  sudo ceph-daemon shell --config ceph.conf --keyring ceph.keyring
+  [any monitor] $ sudo ceph-daemon shell --config ceph.conf --keyring ceph.keyring
```

Comment:
this relies on teh ceph.conf and ceph.keyring files being in the local directory.. but it's the bootstrap step that writes them

Reply:
I've changed this to "node 1".